src/cms/templates/pages/page_tree.html

```diff
@@ -129,6 +115,14 @@ <h2 class="heading font-normal mb-2">{% trans 'Pages' %}</h2>
             </div>
             <input id="bulk-action-execute" type="submit" class="inline-block bg-gray-500 cursor-not-allowed h-full text-white font-bold py-2 px-4 ml-2 rounded" value="{% trans 'Execute' %}" />
         </div>
+        <div class="flex flex-wrap w-1/2 justify-end">
+            <a href="{% url 'archived_pages' region_slug=region.slug language_slug=language.slug %}" class="font-bold text-xs text-gray-800 block pb-2 hover:underline">
+                {% trans 'Archived pages' %}
+                <span class="inline-block rounded-full bg-integreat-500 text-gray-800 px-2 py-1 text-xs font-bold">
+                    {{ archived_count }}
+                </span>
+            </a>
+        </div>        
```

Comment:
![PR #766](https://user-images.githubusercontent.com/16021269/114702756-c2f2af00-9d24-11eb-92e2-d0f10ca05a7d.png)
In this case as well, I think I'd prefer the original layout:
![develop](https://user-images.githubusercontent.com/16021269/114702760-c423dc00-9d24-11eb-893a-0494101678fe.png)